newdle/api.py

```diff
@@ -291,6 +291,25 @@ def get_newdles_participating():
     return NewdleParticipantSchema(many=True).jsonify(newdle)
 
 
+@api.route('/newdle/<code>/create-event', methods=('POST',))
+def create_event(code):
+    newdle = Newdle.query.filter_by(code=code).first_or_404(
+        'Specified newdle does not exist'
+    )
```

Comment:
I think we can require that the user is the creator of the newdle. No harm in any case, but feels dirty to not restrict this API.

Reply:
Updated the query to `Newdle.query.filter_by(code=code, creator_uid=g.user['uid'])...`.

Reply:
explicit check and raising `Forbidden` would be cleaner IMO. i think we do that in other places as well...